src/main/kotlin/no/nav/familie/ks/sak/api/dto/BrevmottakerDto.kt

```diff
@@ -25,3 +25,18 @@ fun BrevmottakerDto.tilBrevMottakerDb(behandlingId: Long) =
         poststed = poststed.trim(),
         landkode = landkode,
     )
+
+fun BrevmottakerDto.harGyldigAdresse(): Boolean {
+    if (landkode == "NO") {
+        return navn.isNotEmpty() &&
+            adresselinje1.isNotEmpty() &&
+            postnummer.isNotEmpty() &&
+            poststed.isNotEmpty()
+    } else {
+        // Utenlandske manuelle brevmottakere skal ha postnummer og poststed satt i adresselinjene
+        return navn.isNotEmpty() &&
+            adresselinje1.isNotEmpty() &&
+            postnummer.isEmpty() &&
+            poststed.isEmpty()
+    }
+}
```

Comment:
```suggestion
fun BrevmottakerDto.harGyldigAdresse(): Boolean =
    if (landkode == "NO") {
        navn.isNotEmpty() &&
                adresselinje1.isNotEmpty() &&
                postnummer.isNotEmpty() &&
                poststed.isNotEmpty()
    } else {
        // Utenlandske manuelle brevmottakere skal ha postnummer og poststed satt i adresselinjene
        navn.isNotEmpty() &&
                adresselinje1.isNotEmpty() &&
                postnummer.isEmpty() &&
                poststed.isEmpty()
    }
```

Kan gjøres om til en single expression siden man fra if else'n returnerer en boolean.